test.js

```diff
@@ -50,6 +50,8 @@ test.afterEach(t => {
 	fs.rmdirSync(t.context.disjoint);
 });
 
+const isNotWindows = process.platform !== 'win32';
```

Comment:
```suggestion
const isWindows = process.platform === 'win32';
```

And use it with `!`.